src/flask_pyoidc/flask_pyoidc.py

```diff
@@ -61,14 +61,14 @@ def __init__(self, provider_configurations, app=None,
         self.clients = None
         self._logout_view = None
         self._error_view = None
+        # current_identity proxy to obtain user info whose token was passed in
+        # the request. It is available until current request only and is
+        # destroyed between the request. The value is set by token_auth
+        # decorator.
+        self.current_identity = LocalProxy(lambda: getattr(_app_ctx_stack.top,
```

Comment:
This looks great - really clean! 💯 

I was thinking about the name; how about `current_token_identity` or just `current_token` to clarify it will only contain data read from an introspected token?

Reply:
I'm good with `current_token_identity`. It seems more descriptive.